### PR TITLE
fix: check `element.content.value` is defined

### PR DESCRIPTION
**Pull request by ajkl2533**
Closes [OACS-5481](https://jira.oraclecorp.com/jira/browse/OACS-5481)

### CODE REVIEW

File: src/Modules/ElementUtils/defaultValue.js [hunk 1]
```diff
@@ -10,11 +11,11 @@ function hasDefault(element) {
     throw new MissingCacheObjectException(element);
   }
 
-  if (isMember(element)) {
-    return element.content.value.cache.hasDefault;
+  if (!isMember(element) || isUndefined(element.content.value)) {
```
Comment (kylef) [inline src/Modules/ElementUtils/defaultValue.js]: I find it is simpler to read without the `!` and `||` and perhaps more logical: If we have a member and the member has a value:

```js
if (isMember(element) && element.content.value) {
  return element.content.value.cache.hasDefault;
}

return element.cache.hasDefault;
```

Feel free to ignore this comment though.

Comment (ajkl2533) [reply src/Modules/ElementUtils/defaultValue.js]: I agree, addressed

File: src/Modules/ElementUtils/defaultValue.js [hunk 2]
```diff
@@ -10,7 +10,7 @@ function hasDefault(element) {
     throw new MissingCacheObjectException(element);
   }
 
-  if (isMember(element)) {
+  if (isMember(element) && element.content.value) {
```
Comment (opichals) [inline src/Modules/ElementUtils/defaultValue.js]:
```suggestion
  if (isMember(element) && hasValue(element)) {
```

File: src/Modules/ElementUtils/type.js [hunk 3]
```diff
@@ -79,6 +80,8 @@ function isMember(element) {
 
 function getTypeOfValue(element) {
   if (isMember(element.element)) {
+    if (isUndefined(element.content.value)) { return 'nullable'; };
```
Comment (opichals) [inline src/Modules/ElementUtils/type.js]:
```suggestion
    if (hasValue(element)) { return 'nullable'; };
```

File: src/Modules/ElementUtils/defaultValue.js (hunk 2 above)
Comment (opichals) [inline src/Modules/ElementUtils/defaultValue.js]:
```suggestion
  if (isMember(element) && !isUndefined(element.content.value)) {
```

**Pull request merged**